src/RemoteResourceS3Controller.js

```diff
@@ -24,6 +24,9 @@ const xml2js = require('xml2js');
 const clone = require('clone');
 const loggerFactory = require('./bunyan-api');
 const { BaseDownloadController } = require('@razee/razeedeploy-core');
+const IamTokenGetter = require('./iam-token');
+
+let iamTokenGetter = new IamTokenGetter()
```

Comment:
I would suggest moving the initialization of the `IamTokenGetter` into the class. Since it's only used in the download function that should be a good place